docs/ops/movement/ShuffleChannels_1.md

```diff
@@ -8,43 +8,57 @@
 
 **Short description**: *ShuffleChannels* permutes data in the channel dimension of the input tensor.
 
+**Detailed description**:
+
+Input ND tensor of `data_shape` is always interpreted as 4D tensor with the following shape:
+
+    dim 0: data_shape[0] * data_shape[1] * ... * data_shape[axis-1] (or 1 if axis == 0)
+    dim 1: group
+    dim 2: data_shape[axis] / group
+    dim 3: data_shape[axis+1] * data_shape[axis+2] * ... * data_shape[data_shape.size()-1] (or 1 if axis points to last dimension)
+
+So the dimensions before and after the dimension pointed by `axis` are flattened and reshaped back to the original shape after channnels shuffling.
+
+
+The operation is equivalent with the following transformation of the input tensor *x* of shape *[N, C, H, W]*:
+
+\f[
+x' = reshape(x, [N, group, C / group, H * W])\\
+x'' = transpose(x', [0, 2, 1, 3])\\
+y = reshape(x'', [N, C, H, W])\\
+\f]
+
+where `group` is the layer attribute described below and the `axis = 1`.
+
 **Attributes**:
 
 * *axis*
 
   * **Description**: *axis* specifies the index of a channel dimension.
-  * **Range of values**: an integer number in the range [-4, 3]
+  * **Range of values**: an integer number in the range `[-rank(data_shape), rank(data_shape) - 1]`
   * **Type**: `int`
   * **Default value**: 1
```

Comment:
Should values also be enclosed between backticks?

Reply:
Do you mean  `**Default value**: 1` or ``[-rank(data_shape), rank(data_shape) - 1]`` ?
I would say the current version is ok. Could you explain your suggestion please?

Reply:
yes I referred to Default value, as for bool types I have seen values true and false surrounded by ``

Reply:
I can see in other ops spec (e.g. [LogSoftmax_5.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/LogSoftmax_5.md) or [Proposal_4.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/detection/Proposal_4.md)) that it's not a rule for numeric values but ofc I can change it, if we decide to be consistent with that.